web/client/utils/__tests__/PluginUtils-test.js

```diff
@@ -72,5 +73,11 @@ describe('PluginsUtils', () => {
         expect(desc1.items[0].cfg).toExist();
 
     });
+    it('combineEpics', () => {
+        const plugins = {MapSearchPlugin: MapSearchPlugin};
+        const appEpics = {appEpics: (actions$) => actions$};
```

Comment:
This sounds like an infinite loop. Is not executed but it should. Please remove it.